conf/docker_test.config

```diff
@@ -0,0 +1,29 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Example Nextflow config file for Travis
+ * -------------------------------------------------
+ */
+params {
+
+  genomes {
+    'docker_test' {
+      fasta   = '/tmp/ngi-rna_test_set/genome.fa'
+      gtf     = '/tmp/ngi-rna_test_set/genes.gtf'
+      bed12   = '/tmp/ngi-rna_test_set/genes.bed'
+      star    = '/tmp/ngi-rna_test_set/star/'
+    }
+  }
+}
+
+docker {
+  enabled = true
+}
+process {
+  container = 'scilifelab/ngi-rnaseq'
+  executor = 'local'
+  cpus = 2
+  memory = '7g'
```

Comment:
I think it's safer to use the nextflow variable types where possible instead of strings - eg `7.GB` instead of `'7g'`